ui/src/app/pages/admin/workspace/basic-information.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Workspace } from 'generated/fetch';
+
+import { isUsingFreeTierBillingAccount } from 'app/utils/workspace-utils';
+
+import { WorkspaceInfoField } from './workspace-info-field';
+
+interface Props {
+  workspace: Workspace;
+}
+export const BasicInformation = ({ workspace }: Props) => (
+  <>
+    <h3>Basic Information</h3>
+    <div className='basic-info' style={{ marginTop: '1.5rem' }}>
+      <WorkspaceInfoField labelText='Workspace Name'>
+        {workspace.name}
+      </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Workspace Namespace'>
+        {workspace.namespace}
+      </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Access Tier'>
+        {workspace.accessTierShortName?.toUpperCase()}
+      </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Google Project Id'>
+        {workspace.googleProject}
+      </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Billing Status'>
+        {workspace.billingStatus}
+      </WorkspaceInfoField>
+      <WorkspaceInfoField labelText='Billing Account Type'>
+        {isUsingFreeTierBillingAccount(workspace)
+          ? 'Initial credits'
```

Comment:
Ok there are some tiny changes like this one: use the updated term instead of the older one